drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_mgmt.c

```diff
@@ -347,15 +347,190 @@ int wifi_nrf_get_power_save_config(const struct device *dev,
 	return ret;
 }
 
+/* TWT interval conversion helpers: User <-> Protocol */
+static struct twt_interval_float wifi_nrf_twt_ms_to_float(uint32_t twt_interval_ms)
+{
+	double mantissa = 0;
+	int exponent = 0;
+	struct twt_interval_float twt_interval_float;
+
```

Comment:
memset twt_interval_float to 0's (in the caller function if the previous comment is acceptable or here if we choose to go without the above suggestion).

Reply:
No need we overwrite in #359, #360.